### DISCUSSION TITLE
Viscosity predict

### DISCUSSION THREAD

**Discussion by Lenpka** [community/community#144168] [Programming Help]
### Body

Trying to teach ML models for Viscosity to predict values using only compositions. But I have a problem - my data is very different. 8314 points - from 1 to 100, the other 3000 points - from 100 to 10000 and more. And these 3000 points include about 300 too big values.
Can you help me?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Nathanael349** [comment]
To handle the disparity and outliers in your viscosity data, try to apply a log transformation to reduce skewness and bring values closer together. Normalize or standardize the data to ensure consistent scaling, then for the extreme values, try to check if they are genuine or real and if not, remove them. Otherwise, use robust modeling techniques that are less sensitive to outliers.
